ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEeaSendTransactionIntegrationTest.java

```diff
@@ -313,7 +313,7 @@ public void signSendTransaction() {
   }
 
   @Test
-  public void missingNonceResultsInNewNonceBeingCreatedAndResent() {
+  public void missingNonceResultsInEthNodeRespondingUnsuccessfully2() {
```

Comment:
this test can probably go - i.e. this is about us doing retries ... which we won't do...